src/main/java/uk/gov/companieshouse/filinghistory/api/controller/FilingHistoryController.java

```diff
@@ -101,7 +101,7 @@ public ResponseEntity<Void> upsertFilingHistoryTransaction(
     @DeleteMapping("/filing-history/{entity_id}/internal")
     public ResponseEntity<Void> deleteFilingHistoryTransaction(@PathVariable("entity_id") final String entityId) {
 
-        DataMapHolder.get().transactionId(entityId);
+        DataMapHolder.get().entityId(entityId);
```

Comment:
Line 123 below, the transactionId can be removed from the message string as its in the log map